lib/tasks/fastboot-server.js

```diff
@@ -120,5 +120,58 @@ module.exports = CoreObject.extend({
         delete require.cache[key];
       }
     });
+  },
+
+  /*
+   * Try to show a useful error message if we're not able to start the user's
+   * app in FastBoot.
+   */
+  printError: function(e) {
```

Comment:
this function looks pretty complicated, it is likely a good idea to have unit tests covering it.